examples/client_example/client_example.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+"""Python Client Example."""
+
+# Copyright 2012 - 2017, New York University and the TUF contributors
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+import argparse
+import logging
+import os
+import shutil
+import sys
+from pathlib import Path
+
+from tuf.exceptions import RepositoryError
+from tuf.ngclient import Updater
+
+# define directory constants
+HOME_DIR = Path.home()  # user home dir
+DOWNLOAD_DIR = "./downloads"  # download dir
+METADATA_DIR = f"{HOME_DIR}/.local/share/tuf_metadata_example"  # metadata dir
+CLIENT_EXAMPLE_DIR = os.path.dirname(os.path.abspath(__file__))  # example dir
+
+
+def init():
+    """
+    The function that initializes the TUF client infrastructure.
+
+    It creates the metadata directory and downloads the ``root.json``.
+    """
+
+    if not os.path.isdir(DOWNLOAD_DIR):
+        os.mkdir(DOWNLOAD_DIR)
+
+    print(f"Download directory [{DOWNLOAD_DIR}] is created")
+
+    if not os.path.isdir(METADATA_DIR):
+        os.makedirs(METADATA_DIR)
+
+    print(f"Metadata folder [{METADATA_DIR}] is created")
+
+    if not os.path.isfile(f"{METADATA_DIR}/root.json"):
+        shutil.copy(
+            f"{CLIENT_EXAMPLE_DIR}/1.root.json", f"{METADATA_DIR}/root.json"
+        )
+        print("Bootstrap initial root metadata")
+
+
+def tuf_updater():
+    """
+    The function that creates the TUF updater using ``tuf.ngclient``.
+    """
+    url = "http://127.0.0.1:8000"
```

Comment:
This is a tricky one... how do we know it's 8000? what if that's in use already?

We can leave solving this to future PRs, but the magic string should not be hidden in this helper function: you have constants at the top of file already, maybe put this there as well for now?

If you define BASE_URL at the top of file, then this function is probably unneeded as it's a single call.